mqttclients/c/mosquitto_client_extensions/mqtt_setup.c

```diff
@@ -10,6 +10,9 @@
 #include "mqtt_callbacks.h"
 #include "mqtt_setup.h"
 
+// A certificate path (any string) is required when configuring mosquitto to use OS certificates.
```

Comment:
It would be good to clarify that this is only needed when specifying a `MQTT_CERT_FILE` and a `MQTT_KEY_FILE` 
```suggestion
// A certificate path (any string) is required when configuring mosquitto to use OS certificates when specifying certificate and key files.
```